### DISCUSSION TITLE
Unload Combine at Percentage

### DISCUSSION THREAD

**Discussion by skidz8426** [Courseplay/Courseplay_FS22#2804] [Ideas]
If I remember in FS19 you could set what percentage to unload a combine at, I haven't seen that here in FS22 CP.  If I have a harvester that holds 16,000L  and a trailer that holds 8,000L  I'd like to set my unload at 50%.  There are other examples where this is useful like a field where harvester fills up like 1 1/2 rows.   maybe even an option where the combine unloads every time it's on an open pass.

**Comment by Tensuko** [comment]
That is a fixed value and won't be changed due to some reasons (calculations and stuff).
